packages/appium/lib/cli/extension-command.js

```diff
@@ -486,4 +487,51 @@ export default class ExtensionCommand {
     this.config.installedExtensions[ext].version = version;
     await this.config.write();
   }
-}
+
+  /**
+   * Runs a script cached inside the "scripts" field under "appium"
+   * inside of the driver/plugins "package.json" file
+   *
+   * @param {string} ext - name of the extension to run a script from
+   * @param {string} scriptName - name of the script to run
+   */
+  async run ({ext, scriptName}) {
+    if (!_.has(this.config.installedExtensions, ext)) {
+      throw new Error(`please install the ${this.type} first`);
+    }
+
+    const extConfig = this.config.installedExtensions[ext];
+
+    if (!_.has(extConfig, 'scripts')) {
+      throw new Error(`The ${this.type} named '${ext}' does not contain the ` +
+                      `"scripts" field underneath the "appium" field in its package.json`);
+    }
+
+    const extScripts = extConfig.scripts;
+
+    if (!_.has(extScripts, scriptName)) {
+      throw new Error(`The ${this.type} named '${ext}' does not support the script: '${scriptName}'`);
+    }
+
+    const runner = new SubProcess(`node`, [extScripts[scriptName]],
+                                  {cwd: this.config.getExtensionRequirePath(ext)});
+
+    const output = [];
+
+    runner.on('stream-line', (line) => {
+      output.push(line);
+      log(this.isJsonOutput, line);
+    });
+
+    await runner.start();
```

Comment:
this might miserably fail if there was no output for more than 5 seconds

Reply:
still not resolved. Please pass a single `0` argument to the `start` call to avoid any checks